app/test/unit/specs/tour.spec.js

```diff
@@ -193,7 +193,7 @@ describe('tour', () => {
   })
   describe('enter-search', () => {
     describe('show', () => {
-      it('should give focus to #search-text and select it\'s contents', () => {
+      it(`should give focus to #search-text and select it's contents`, () => {
```

Comment:
Should be "its," not "it's" (thus eliminating the quote-escape problem).

Reply:
I will change it, so its correct.  ;)  (did that one on purpose)